src/index.html

```diff
@@ -22,6 +22,410 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="#"
+      >
+        <img
+          class="moyo_logo"
+          src="images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              apple
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="#"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="#"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
```

Comment:
The `data-qa` attribute is used here, but it is not consistently applied across all navigation links. Ensure that the `data-qa` attribute is used consistently if it's intended for testing or automation purposes.